README.md

```diff
@@ -70,11 +70,11 @@ console.log(icon);
 */
 ```
 
-Alternatively you can import the needed icons individually.
+Alternatively you can import the needed icons individually, where `[ICON NAME]` is replaced by the [icon slug].
```

Comment:
Given that we have now moved away from passing the icon's title to `simpleIcons.get()` in favour of the slug, should we not update the example code to use `[ICON SLUG]` instead of `[ICON NAME]` allowing us to simplify these lines and avoid any potential confusion?

Reply:
I'd be in favour of that, though I still think we need some text to link to the table :+1:

Reply:
Oh, definitely 👍🏻